Day5_excercise_1.rb

```diff
@@ -0,0 +1,39 @@
+#1.Create an array which will store the square of each value between 2 and 5, inclusive.
+list = []
+(2..5).each do |i|
+  print list.push(i * i)
```

Comment:
I don't think you print out the correct thing. Can you copy / paste your answer for question 1?